apps/easypid/app.config.js

```diff
@@ -1,182 +1,58 @@
 const { version } = require('./package.json')
+const { createBaseConfig } = require('./base.app.config')
 
-const APP_VARIANT = process.env.APP_VARIANT || 'production'
-
-const variants = {
-  development: {
-    bundle: '.dev',
-    name: ' (Dev)',
-    mediatorDid: 'did:web:mediator.dev.paradym.id',
-  },
-  preview: {
-    bundle: '.preview',
-    name: ' (Preview)',
-    mediatorDid: 'did:web:mediator.paradym.id',
-  },
-  production: {
-    bundle: '',
-    name: '',
-    mediatorDid: 'did:web:mediator.paradym.id',
-  },
+const mediatorDids = {
+  development: 'did:web:mediator.dev.paradym.id',
+  preview: 'did:web:mediator.paradym.id',
+  production: 'did:web:mediator.paradym.id',
 }
 
-const variant = variants[APP_VARIANT]
-
-if (!variant) {
-  throw new Error(`Invalid variant provided: ${process.env.APP_VARIANT}`)
+const APP_CONFIGS = {
+  FUNKE_WALLET: createBaseConfig({
+    name: 'Funke Wallet',
+    scheme: 'id.animo.ausweis',
+    icon: './assets/funke/icon.png',
+    adaptiveIcon: './assets/funke/adaptive-icon.png',
+    splash: './assets/funke/splash.png',
+    slug: 'ausweis-wallet',
+    version,
+    bundleId: 'id.animo.ausweis',
+    additionalInvitationSchemes: ['id.animo.ausweis', 'haip'],
+    associatedDomains: ['funke.animo.id'],
+    projectId: '28b058bb-3c4b-4347-8e72-41dfc1dd99eb',
+  }),
+
+  PARADYM_WALLET: createBaseConfig({
+    name: 'Paradym Wallet',
+    scheme: 'id.animo.paradym',
+    icon: './assets/paradym/icon.png',
+    adaptiveIcon: './assets/paradym/adaptive-icon.png',
+    splash: './assets/paradym/splash.png',
+    slug: 'paradym-wallet',
+    version,
+    bundleId: 'id.animo.paradym',
+    additionalInvitationSchemes: ['didcomm'],
```

Comment:
This should use the same additional schemes as easypid (plus didcomm)

Reply:
Also did you address the hardcoded id.animo.ausweis in the app? (I think i shared this in slack)